.github/workflows/test.yml

```diff
@@ -29,10 +31,18 @@ jobs:
         with:
           python-version: ${{ matrix.python }}
 
+      # NOTE: We install in both editable, and non-editable mode. The former is
+      # necessary to produce coverage reports with the correct numbers.
+      # https://github.com/cylc/cylc-uiserver/issues/179
       - name: Install cylc-uiserver & dependencies
         run: |
           pip install git+https://github.com/cylc/cylc-flow@master
-          pip install .[all]
+          if [ "${{ matrix.python }}" == "${{ env.PYTHON_COVERAGE_ENABLED }}" ]; then
+            pip install -e .[all]
+          else
+            pip install .[all]
+          fi
```

Comment:
Installing in editable mode only when we want to run coverage, and in non-editable mode otherwise. This way we end up testing both? I think we had an issue some time ago where something worked in editable mode, and not in the default mode. Can't remember if it was relative imports, or some CI stuff…